init.sql

```diff
@@ -159,3 +159,128 @@ create table asset_creators
+    name_length                int,                                     
+    prefix_uri                 varchar(10),                            
+    uri_length                 int,                                    
+    is_sequential              bool,
+    gatekeeper_network         bytea,                                    
+    expire_on_use              bool,  
+    number                     int,                                     
+    end_setting_type           end_setting_type,    
+    name                       varchar(50),                              
+    uri                        varchar(200),                             
+    hash                       bytea                                                                    
+);
+create unique index candy_machine_data_candy_machine_id on candy_machine_data (candy_machine_id);
+
+create table candy_machine_creators
+(
+    id                    bigserial                                PRIMARY KEY,
+    candy_machine_id      bytea references candy_machine (id)      not null,
+    creator               bytea                                    not null,
+    share                 int                                      not null default 0,
+    verified              bool                                     not null default false
+);
+create unique index candy_machine_creators_candy_machine_id on candy_machine_creators (candy_machine_id);
+create index candy_machine_creator on candy_machine_creators (candy_machine_id, creator);
+create index candy_machine_verified_creator on candy_machine_creators (candy_machine_id, verified);
+
+create table candy_guard
+(   
+    id                   bytea                                   PRIMARY KEY,
+    bump                 int                                     not null,
+    authority            bytea                                   not null,
+)
+
+create table candy_guard_group
+(
+    id                   bigserial                              PRIMARY KEY,
+    label                varchar(50),
+    candy_guard_id       bytea references candy_guard (id)      not null,
+    mode                 whitelist_mint_mode,                                      
+    whitelist_mint       bytea,                                                    
+    presale              bool,                                                    
+    discount_price       int,
+    lamports             int,
+    last_instruction     bool,
+    amount               int,
+    destination          bytea,
+    spl_token_amount     int,                                    
+    token_mint           bytea,                        
+    destination_ata      bytea,         
+    date                 int,    
+    signer_key           bytea,    
+    burn                 bool,                        
+    required_collection  bytea,        
+    merkle_root          bytea,      
+    limit                int,     
```

Comment:
For mint limit there's also an ID that would be separate from your normal ID in this table.

Reply:
this was a miss on my part, adding now